react-exercises/modal-react/src/App.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import "./App.css";
+import LoginModal from "./component/LoginModal";
+import SignUpModal from "./component/SignUpModal";
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
In constructor less component you can use state without "this".

Reply:
Will there be any specific benifit we get ? it's usually better for a component to manage it's own state and lifecycle.
We use callbacks for message passing.

Reply:
you can manage state and lifecycle without using constructor too. benefit is that it looks simpler and easy code to understand.